hardhat/hardhat-exposed/internal/expose.ts

```
@@ -0,0 +1,725 @@
+}
+
+function getAbiType(
+  typeName: TypeName,
+  context: ContractDefinition,
+  deref: ASTDereferencer,
+  location: StorageLocation | null,
+): string {
+  switch (typeName.nodeType) {
+    case 'ElementaryTypeName':
+    case 'ArrayTypeName':
+      const { typeString } = typeName.typeDescriptions;
```

Biome (2.4.15)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)